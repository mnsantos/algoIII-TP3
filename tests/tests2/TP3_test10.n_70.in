70 778
3 10
4 10
1 2
1 8
6 10
1 7
7 10
1 3
7 9
2 3
1 4
4 11
7 12
1 16
1 17
5 8
1 9
7 14
8 12
1 5
1 10
5 7
3 4
2 9
1 11
1 15
1 14
6 9
3 11
1 18
3 9
6 13
1 6
1 12
5 9
5 6
2 11
4 6
4 12
2 13
10 12
3 6
68 69
66 69
65 70
65 69
65 67
64 66
63 65
62 67
62 64
19 65
19 57
19 55
19 52
19 38
19 36
19 35
19 34
19 29
19 23
19 22
19 20
18 70
18 69
18 57
18 44
18 43
18 38
18 35
18 33
18 30
18 29
17 70
17 69
17 64
17 61
17 59
17 58
17 53
17 50
17 46
17 45
17 43
17 42
17 36
17 34
17 31
17 28
17 27
17 26
17 23
17 21
17 20
17 18
16 69
16 68
16 67
16 65
16 63
16 60
16 59
16 55
16 54
16 49
16 48
16 47
16 45
16 42
16 38
16 34
16 23
16 20
15 68
15 67
15 64
15 61
15 60
15 56
15 52
15 46
15 45
15 37
15 30
15 29
15 24
15 22
15 20
14 69
14 64
14 63
14 56
14 53
14 52
14 51
14 49
14 48
14 42
14 40
14 39
14 30
14 29
14 24
14 23
14 22
14 19
13 69
13 67
13 64
13 62
13 56
13 55
13 54
13 45
13 38
13 37
13 36
13 27
13 26
13 25
13 24
13 19
13 16
12 68
12 62
12 57
12 55
12 53
12 51
12 50
12 49
12 48
12 44
12 42
12 41
12 35
12 31
12 30
12 26
12 25
12 20
12 18
12 14
11 70
11 67
11 64
11 63
11 55
11 53
11 48
11 45
11 36
11 35
11 31
11 30
11 28
11 26
11 22
11 17
10 70
10 67
10 64
10 61
10 58
10 52
10 49
10 48
10 43
10 42
10 39
10 38
10 31
10 28
10 24
10 22
10 15
10 14
9 65
9 64
9 63
9 45
9 44
9 43
9 41
9 37
9 35
9 34
9 32
9 25
9 24
9 19
8 70
8 64
8 63
8 54
8 53
8 52
8 51
8 45
8 40
8 34
8 32
3 26
3 25
3 22
2 65
2 62
2 61
2 56
2 54
2 53
2 51
2 46
2 38
2 36
2 33
2 28
2 27
2 25
2 24
2 20
1 70
1 69
1 68
1 67
1 66
1 65
1 64
1 63
1 62
1 61
1 60
1 59
1 58
1 57
1 56
1 55
1 54
1 53
1 52
1 51
1 26
1 25
1 24
1 23
1 22
1 21
1 20
1 19
1 27
1 28
1 29
1 30
1 31
1 32
1 33
1 34
1 35
1 36
1 37
1 38
1 39
1 40
1 41
1 42
1 43
1 44
1 45
1 46
1 47
1 48
1 49
1 50
3 28
3 30
3 32
3 34
3 38
3 40
3 41
3 42
3 47
3 48
3 50
3 51
3 52
3 56
3 57
3 58
3 59
3 62
3 64
3 66
3 68
4 20
4 28
4 30
4 32
4 35
4 37
4 46
4 48
4 52
4 53
4 54
4 64
4 67
5 26
5 31
5 33
5 37
5 43
5 51
5 52
5 55
5 60
5 65
5 70
6 16
6 17
6 24
6 31
6 33
6 41
6 43
6 46
6 48
6 49
6 51
6 53
6 54
6 62
7 16
7 24
7 27
7 32
7 33
7 35
7 42
7 43
7 44
7 47
7 49
7 52
7 56
7 58
7 66
8 16
8 22
8 26
8 27
8 30
19 66
19 69
20 23
20 25
20 26
20 36
20 40
20 41
20 48
20 50
20 51
20 52
20 54
20 57
20 58
20 64
20 66
21 26
21 27
21 30
21 34
21 35
21 47
21 50
21 51
21 52
21 54
21 56
21 58
21 65
22 23
22 26
22 28
22 29
22 33
22 34
22 35
22 39
22 45
22 49
22 51
22 52
22 57
22 58
22 59
22 62
22 63
22 68
22 69
23 25
23 27
23 28
23 29
23 30
23 37
23 39
23 40
23 42
23 49
23 50
23 51
23 56
23 63
24 25
24 27
24 30
24 35
24 38
24 39
24 48
24 54
24 57
24 59
24 66
24 69
25 28
25 29
25 32
25 33
25 38
25 41
25 45
25 57
25 59
25 65
25 66
25 67
25 69
26 27
26 32
26 34
26 35
26 37
26 40
26 41
26 43
26 49
26 55
26 57
26 59
26 63
26 64
26 70
27 28
27 30
27 35
27 37
27 44
27 47
27 50
27 52
27 56
27 63
27 70
28 33
28 37
28 38
28 41
28 54
28 57
28 59
28 64
28 66
28 69
28 70
29 34
29 39
29 42
29 48
29 55
29 60
29 65
30 39
30 43
30 46
30 49
30 50
30 52
30 58
31 37
31 40
31 41
31 43
31 47
31 48
31 50
31 52
31 53
31 55
31 57
31 58
31 59
31 60
31 61
31 63
31 64
31 68
31 70
32 34
32 35
32 39
32 41
32 43
32 44
32 46
32 50
32 55
32 59
32 60
32 64
32 65
32 68
32 69
32 70
33 35
33 38
33 41
33 51
33 58
33 68
34 41
34 42
34 45
34 49
34 53
34 56
34 58
34 63
34 64
34 65
35 41
35 42
35 47
35 50
35 56
35 62
35 64
36 37
36 44
36 49
36 50
36 51
36 54
36 55
36 59
36 61
36 64
36 67
37 38
37 47
37 48
37 49
37 53
37 54
37 56
37 60
37 67
37 68
37 69
38 43
38 44
38 48
38 52
39 43
39 46
39 47
39 50
39 57
39 62
39 65
39 69
40 51
40 52
40 56
40 57
40 58
40 59
40 65
41 42
41 46
41 47
41 51
41 53
41 57
41 59
41 63
41 67
41 69
41 70
42 48
42 49
42 51
42 53
42 56
42 58
42 66
42 67
42 68
42 69
43 48
43 51
43 55
43 58
43 60
43 61
43 64
43 65
43 67
44 49
44 50
44 53
44 55
44 57
44 59
44 62
44 63
44 65
44 67
44 68
45 49
45 53
45 61
45 64
45 67
45 68
45 69
46 51
46 53
46 54
46 57
46 58
46 63
46 64
46 67
46 70
47 49
47 50
47 54
47 59
47 61
47 62
47 64
47 65
47 66
47 67
47 69
48 49
48 50
48 53
48 54
48 57
48 58
48 59
48 64
48 67
48 69
49 53
49 57
49 58
49 59
49 63
49 68
50 51
50 53
50 55
50 60
50 62
50 67
50 68
50 70
51 52
51 53
51 55
51 56
51 59
51 64
51 65
51 69
51 70
52 53
52 63
52 70
53 56
53 57
53 58
53 60
53 62
53 65
54 55
54 56
54 57
54 61
54 63
54 64
54 66
54 69
55 60
55 63
55 65
55 66
56 61
56 65
56 66
56 68
56 69
56 70
57 58
57 60
57 64
57 65
58 60
58 62
58 64
58 66
59 60
59 62
59 68
59 69
60 61
60 62
60 63
60 64
61 62
61 68
1 13
5 16
0
