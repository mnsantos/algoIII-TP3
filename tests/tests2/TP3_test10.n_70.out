182 7 1 10 31 43 48 58 64
